bin/crew

```diff
@@ -724,12 +724,12 @@ def download
       # Recall repository from cache if requested
       if CREW_CACHE_ENABLED
         # No git branch specified, just a git commit or tag
-        if @pkg.git_branch.nil? || @pkg.git_branch.empty?
-          abort("No Git branch, commit, or tag specified!").lightred if @pkg.git_hashtag.nil? || @pkg.git_hashtag.empty?
+        if @pkg.git_branch.to_s.empty?
+          abort("No Git branch, commit, or tag specified!").lightred if @pkg.git_hashtag.to_s.empty?
           cachefile = CREW_CACHE_DIR + filename + @pkg.git_hashtag.gsub('/', '_') + '.tar.xz'
           puts "cachefile is #{cachefile}".orange if @opt_verbose
         # Git branch and git commit specified
-        elsif ( ! @pkg.git_hashtag.nil? || ! @pkg.git_hashtag.empty )  and ( ! @pkg.git_branch.nil? || ! @pkg.git_branch.empty?)
+        elsif ! @pkg.git_hashtag.to_s.empty? and ! @pkg.git_branch.to_s.empty?
```

Comment:
`@pkg.git_branch.to_s.empty?` already checked above?

Reply:
That's looking for `! @pkg.git_branch.to_s.empty?`, so the reverse condition...